src/pages/Profile.tsx

```diff
@@ -257,12 +290,14 @@ const Profile = () => {
                     </Box>
                   )}
                 </Box>
+
                 <Box sx={styles.followerContainer}>
                   <Link
                     color={theme.palette.black.main}
                     component={Routerlink}
                     to={`/${username}`} // TODO: Create Modal to check followers
```

Comment:
Remove this comment and the one on line 310 too please

Reply:
Removed outdated comments